symLinkFeatures.ps1

```diff
@@ -0,0 +1,29 @@
+# To run this you will need to ensure a few things first:
+# 1. temporarily change the execution policy: Set-ExecutionPolicy Unrestricted -Scope Process
+# 2. download and extract the latest dev-containers release from https://github.com/microsoft/vscode-dev-containers/releases/
+# 3. Rename the root folder of that extracted release from "package" to "vscode-dev-containers" 
+
+
+# Run the script in powershell with the following three arguments:
+# ./symLinkFeatures.ps1 "full/path/to/vscode-dev-containers-repo" "full/path/to/vscode-dev-containers" "full/path/to/vscode-remote-containers"
```

Comment:
could you add a small section explaining the vscode-dev-containers-repo vs full/path/to/vscode-dev-containersand why that is needed for this script? // for documentation purposes if others may need to run this script.